moveit_ros/move_group/src/default_capabilities/query_planners_service_capability.cpp

```diff
@@ -73,94 +73,98 @@ void MoveGroupQueryPlannersService::initialize()
-      planner_interface->getPlanningAlgorithms(pi_desc.planner_ids);
-      res->planner_interfaces.push_back(pi_desc);
-    }
-  }
-  return true;
+  // TODO(sjahr): This is currently not working. Need to decide whether to fix or remove this.
+  // for (const auto& planning_pipelines : context_->moveit_cpp_->getPlanningPipelines())
+  // {
+  //   const auto& pipeline_id = planning_pipelines.first;
+  //   const auto& planning_pipeline = planning_pipelines.second;
+  //   const planning_interface::PlannerManagerPtr& planner_interface = planning_pipeline->getPlannerManager();
+  //   if (planner_interface)
+  //   {
+  //     std::vector<std::string> algs;
+  //     planner_interface->getPlanningAlgorithms(algs);
+  //     moveit_msgs::msg::PlannerInterfaceDescription pi_desc;
+  //     pi_desc.name = planner_interface->getDescription();
+  //     pi_desc.pipeline_id = pipeline_id;
+  //     planner_interface->getPlanningAlgorithms(pi_desc.planner_ids);
+  //     res->planner_interfaces.push_back(pi_desc);
+  //   }
+  // }
+  return false;
 }
 
-bool MoveGroupQueryPlannersService::getParams(const std::shared_ptr<rmw_request_id_t>& /* unused */,
-                                              const std::shared_ptr<moveit_msgs::srv::GetPlannerParams::Request>& req,
-                                              const std::shared_ptr<moveit_msgs::srv::GetPlannerParams::Response>& res)
+bool MoveGroupQueryPlannersService::getParams(
+    const std::shared_ptr<rmw_request_id_t>& /* unused */,
+    const std::shared_ptr<moveit_msgs::srv::GetPlannerParams::Request>& /* unused */,
+    const std::shared_ptr<moveit_msgs::srv::GetPlannerParams::Response>& /* unused */)
 {
-  const planning_pipeline::PlanningPipelinePtr planning_pipeline = resolvePlanningPipeline(req->pipeline_id);
-  if (!planning_pipeline)
-    return false;
-
-  const planning_interface::PlannerManagerPtr& planner_interface = planning_pipeline->getPlannerManager();
-  if (planner_interface)
-  {
-    std::map<std::string, std::string> config;
-
-    const planning_interface::PlannerConfigurationMap& configs = planner_interface->getPlannerConfigurations();
-
-    planning_interface::PlannerConfigurationMap::const_iterator it =
-        configs.find(req->planner_config);  // fetch default params first
-    if (it != configs.end())
-      config.insert(it->second.config.begin(), it->second.config.end());
-
-    if (!req->group.empty())
-    {  // merge in group-specific params
-      it = configs.find(req->group + "[" + req->planner_config + "]");
-      if (it != configs.end())
-        config.insert(it->second.config.begin(), it->second.config.end());
-    }
-
-    for (const auto& key_value_pair : config)
-    {
-      res->params.keys.push_back(key_value_pair.first);
-      res->params.values.push_back(key_value_pair.second);
-    }
-  }
-  return true;
+  // TODO(sjahr): This is currently not working. Need to decide whether to fix or remove this.
```

Comment:
same here, where is this and `setParams()` used currently?

Reply:
See above. This needs to be updated